Deploy/uwp_jcadmu.ps1

```diff
@@ -638,72 +758,190 @@ if (Get-Item $ADMUKEY -ErrorAction SilentlyContinue) {
     $ftaOutput = @()
     $ptaOutput = @()
 
-    # Create progress bar for $appxList $ftalist and $ptalist
-    # Create a foreach loop for each list and do a percent even if it is < 100
-    $allListsCount = $appxList.Count + $ftaList.Count + $ptaList.Count
+    Write-ToLog -Message ("Begin Appx File Registration")
+    Write-ToLog -Message ("There are $($appxList.count) apps to be registered")
+    $logFile = "$HOME\AppData\Local\JumpCloudADMU\appx_statusLog.txt"
+    $homepath = $HOME
+    # Remove existing log file to ensure a fresh start.
+    if (Test-Path $logFile) {
+        Remove-Item $logFile -Force
+    }
+
+    # TODO: IF appx is null do not start
+    $j = Start-Job -ScriptBlock {

```

Comment:
When the UWP tool starts this job will always run to import the Appx Files. TODO: additional edge cases and timeout

Reply:
Added the timeout functionality for this